config/settings.yml.example

```diff
@@ -6,12 +6,11 @@ DEFAULT_PREFIX:
 
 USER_IDS:
   SYSADMINS:
-    - 123456789012345679
-    - 123456789012345679
-  BOT_OWNER: 123456789012345679
+    - 123456789012345
```

Comment:
any reason why these were changed

Reply:
Accidentally moved instead of copied the .example file
Before committing I recreated a .example file.